tests/auto/validate_prompts.test.js

```
@@ -0,0 +1,861 @@
+#!/usr/bin/env node
+
+/**
+ * Comprehensive tests for PromptValidator class
+ * Tests all public methods with happy path and edge cases
+ */
+
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)